firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/common/shared/Types.kt

```diff
@@ -84,7 +86,7 @@ internal data class Blob(@SerialName("mime_type") val mimeType: String, val data
 
 @Serializable internal data class CodeExecutionResult(val outcome: Outcome, val output: String)
 
-@Serializable
+@Serializable(OutcomeSerializer::class)
 internal enum class Outcome {
   @SerialName("OUTCOME_UNSPECIFIED") UNSPECIFIED,
```

Comment:
Should this be `UNKNOWN`?

Reply:
Actually, it should be gone, since it's not used :-)